src/arrays/creating_arrays/hard/q1/RandomArray.java

```diff
@@ -0,0 +1,42 @@
+package arrays.creating_arrays.hard.q1;
+
+/*
+Write a Java program that asks the user for the number of elements it will store and uses a method called
```

Comment:
Change question text to:
Write a Java program that asks the user to enter a length for an array and then calls a method named createRandomArray() to generate this array. The method should accept an integer for the length and create a new array of that size. Then, it should initialize the array with random numbers from 0 (inclusive) to the passed length (exclusive). Return this array and print the output in the main method.